tests/playwright/sanity/modules/nested-accordion/helper.js

```diff
@@ -0,0 +1,109 @@
+const { expect } = require( '@playwright/test' );
+
+/**
+ * Set Nested Accordion Title Tag (H1-H6,div,span,p)
+ *
+ * @param {string} optionToSelect          - value of select option i.e. h1,h2,h3,h4,h5,h6,div,span,p
+ * @param {string} nestedAccordionWidgetId - id of the nested accordion widget
+ * @param {Object} editor
+ * @param {Object} page
+ * @return {Promise<void>}
+ */
+async function setTitleTextTag( optionToSelect, nestedAccordionWidgetId, editor, page ) {
+	const frame = editor.getPreviewFrame();
+	await editor.selectElement( nestedAccordionWidgetId );
+	await page.selectOption( '.elementor-control-title_tag .elementor-control-input-wrapper > select', optionToSelect );
+	await frame.waitForLoadState( 'load' );
+}
+
+/**
+ * Take a Screenshot of this Widget
+ *
+ * @param {string} fileName
+ * @param {Object} locator
+ * @return {Promise<void>}
+ */
+async function expectScreenshotToMatchLocator( fileName, locator ) {
+	expect.soft( await locator.screenshot( {
+		type: 'png',
+	} ) ).toMatchSnapshot( fileName );
```

Comment:
@Barcov Is this test flaky? It failed once.

Reply:
@louiswol94 This is essentially shorthand for a normal screenshot with png and soft expect dialled in. So could only be flaky if the locator is flaky. 

We'll possibly need to increase the retires for isUIStable() on the failing screenshot if we see this again.